docs/source/tutorial/package_a_project.md

```diff
@@ -4,25 +4,96 @@ This section explains how to build your project documentation, and how to bundle
 
-Once you have added any documentation you need, run the following from the project root directory:
+Navigate to the empty `docs` directory in the terminal, and run the following command:
 
 ```bash
-kedro build-docs --open
+sphinx-quickstart
 ```
 
-The HTML documention is built to `docs/build/html` and opens automatically in a browser tab.
+Sphinx will ask a series of configuration questions. The first is as follows:
 
-```{note}
-The `build-docs` command creates documentation based on the code structure of your project. Documentation includes any [`docstrings`](https://datacamp.com/community/tutorials/docstrings-python) defined in your code.
+```text
+You have two options for placing the build directory for Sphinx output.
+Either, you use a directory "_build" within the root path,
+or you separate "source" and "build" directories within the root path.
+
+> Separate source and build directories (y/n)? [n]:
+```
+
+In the following instructions, we assume that you select `y` to separate the built files from the source files, but you can choose either option safely and continue to follow the process.
+
+Enter any additional information that Sphinx requests such as the project name and the documentation language (which defaults to English).
+
+
+### Build HTML documentation
+When the quickstart process is complete, you may add project-specific markdown documentation within the `docs/source` folder of your Kedro project and update `docs/source/index.rst` to add it to the table of contents.
+
+To build the documentation, run the following **from the `docs` directory**:
+
+```bash
+make html
+```
+
+Your project documentation is built to the `docs/build/html` directory.
+
+### Documentation from docstrings
+If you wish to add documentation built from [`docstrings`](https://datacamp.com/community/tutorials/docstrings-python) within your project, you need to make the following changes to the Sphinx configuration files found in the `docs/source` directory to use [automatic documentation generation from code](https://www.sphinx-doc.org/en/master/tutorial/automatic-doc-generation.html).
+
+Add the following to the beginning of the `docs/source/conf.py` file:
+
+```python
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join("..", "..", "src")))
+```
+
+Also in `conf.py`, add the following to ensure that the `sphinx.ext.autodoc` and `sphinx.ext.autosummary` extensions are specified, and `autosummary_generate` is enabled:
+
+```python
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.autosummary"]
+autosummary_generate = True
+```
+
+Finally, to ensure that you include the autodoc modules in your build, create a separate file called `api.rst` in `docs/source`. You must replace `project_name` with the name of your project:
+
+```text
+
+.. autosummary::
```

Comment:
I might have done something wrong here, but I don't think `.. autosummary::` of the project generates the stubs automatically. Instead, I'm getting an empty page. This behavior still confuses me a lot and it's probably the reason why folks created https://github.com/rdb/sphinx-autopackagesummary, https://github.com/readthedocs/sphinx-autoapi, https://github.com/astropy/sphinx-automodapi, and many others.

In any case, this is a deviation from what `kedro build-docs` currently does (uses `sphinx-apidoc` instead of autosummary) so perhaps it's not worth changing that here? Explaining folks how to effectively use the already difficult Sphinx API is probably out of scope...

Reply:
Thanks for the revisions @astrojuanlu I think I get it now -- I'll run through on an empty project just to confirm it works for me and that query about whether to `pip install` the project.

Reply:
You're right, I forgot to do that. Looks perfect now!